pact/coin-contract/coin.pact

```diff
@@ -6,19 +6,28 @@
   \create. To access the coin contract, you may use its fully-qualified name,  \
   \or issue the '(use coin)' command in the body of a module declaration."
 
-  @model [
-    (defproperty conserves-mass
-      (= (column-delta coin-table 'balance) 0.0))
-  ]
-
-  (implements coin-sig)
+  @model
+    [ (defproperty conserves-mass
+        (= (column-delta coin-table 'balance) 0.0))
+
+      (defproperty account-structure (account:string)
+        (and
+          (!= account "")
+          (and
+            (>= (length account) 3)
+            (<= (length account) 256))))
```

Comment:
If the length is `>= 3` then it can't be empty, so the `!=` check is redundant.

Reply:
Ah true!